### PR TITLE
Fixed handling raw data from HMI

### PR DESCRIPTION
**Pull request by KKarlash**

### CODE REVIEW

File: src/components/vr_cooperation/src/command/process_data_request.cc [hunk 1]
```diff
@@ -55,6 +55,20 @@ ProcessDataRequest::~ProcessDataRequest() {
 void ProcessDataRequest::Execute() {
   LOG4CXX_AUTO_TRACE(logger_);
   application_manager::MessagePtr message_to_send;
+  vr_hmi_api::ProcessDataRequest params;
+  if (gpb_message().has_params()
+      && params.ParseFromString(gpb_message().params())) {
+    std::vector<uint8_t> *binary_data = NULL;
+    if (params.has_raw()) {
+      binary_data = new std::vector<uint8_t>();
```
Comment (KKarlash) [inline src/components/vr_cooperation/src/command/process_data_request.cc]: memory will be freed inside of application_manager::MessagePtr message

**Comment by KKarlash** [comment]
@bonewell 
@VladSemenyuk 
@DKlimenko 
@giangtqh 
@thinhts

File: src/components/vr_cooperation/src/command/process_data_request.cc [hunk 2]
```diff
@@ -55,6 +55,20 @@ ProcessDataRequest::~ProcessDataRequest() {
 void ProcessDataRequest::Execute() {
   LOG4CXX_AUTO_TRACE(logger_);
   application_manager::MessagePtr message_to_send;
+  vr_hmi_api::ProcessDataRequest params;
+  if (gpb_message().has_params()
+      && params.ParseFromString(gpb_message().params())) {
```
Comment (bonewell) [inline src/components/vr_cooperation/src/command/process_data_request.cc]: It will be good to see message about parsing issue in the logs.

**Comment by DKlimenko** [comment]
reviewed

**Pull request merged**